api/routes/accountBalances.js

```diff
@@ -88,7 +88,9 @@ var accountBalances = function (req, res, next) {
     if (err.code === 400) {
```

Comment:
this could be simplified with something like: 
```
if (err.code.toString()[0] === '4') {
  log.error(err.error || err);
  response.json({result:'error', message:err.error}).status(err.code).pipe(res);
}
```
since you do basically the same thing in either case